### ISSUE TITLE
This User is a Group

### ISSUE THREAD

**Issue by ch0ccyra1n** [TheEssem/mastodon#10]
### Pitch

I think it would be great to be able to designate an account as a Group.

### What this would actually do:

1. Change the type of ActivityStreams `Actor` from `Person` to `Group`
2. Automatically boost posts that mention the group (Or alternatively this could be a separate box to tick)

### Motivation

I think this feature would be useful for those who want to create a group but would like to stay on a chuckya instance with a separate group account (like me)

**Comment by TheEssem** [comment]
There's a PR for group support upstream (and it's shown as "in progress" on the [roadmap](https://joinmastodon.org/roadmap)), however it's still unfinished and seems to be stalled: mastodon/mastodon#19059

I'd personally rather wait until upstream's implementation is complete (if possible), since I want to avoid a situation where we have our own, incompatible implementation and have to figure out what to do with it once upstream merges theirs. Seeing as it hasn't been worked on in over a year, however, I wonder if it would be possible to find a way to finish it up ourselves and contribute the changes upstream (though that would be difficult considering that we'd have to port it to glitch-soc and rebase everything onto a codebase that's changed heavily)...

**Comment by ski-freak** [comment]
Yeah it would be really nice if someone contributes to the upstream implementation and finishes it.

**Comment by Leatherface75** [comment]
This shows how Friendica have done this.

![image](https://github.com/TheEssem/mastodon/assets/2489019/92c73839-890b-4586-8f31-af2427af2d49)

**Comment by Sqaaakoi** [comment]
> This shows how Friendica have done this.
> 
> ![image](https://private-user-images.githubusercontent.com/2489019/293347843-92c73839-890b-4586-8f31-af2427af2d49.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQxNjEwMzEsIm5iZiI6MTcwNDE2MDczMSwicGF0aCI6Ii8yNDg5MDE5LzI5MzM0Nzg0My05MmM3MzgzOS04OTBiLTQ1ODYtOGYzMS1hZjI0MjdhZjJkNDkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDEwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAxMDJUMDE1ODUxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NzAxNTk2NDYzZmNkMWE0ZDA2YTA2ZGQ4OWI3ZGZhNWRhY2ZlM2VjYjMwM2UzYjI1Njc0YzQ2MDRmMzVkNmViYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.gLG9p3k2qVsgGzJRur7xV4yGC34BueVu8tv8fkaVWV0)

That looks very overwhelming and needlessly descriptive
